builtin/logical/aws/secret_access_keys.go

```diff
@@ -48,42 +47,25 @@ func secretAccessKeys(b *backend) *framework.Secret {
 }
 
 func genUsername(displayName, policyName, userType, usernameTemplate string) (ret string, warning string, err error) {
-	switch userType {
-	case "iam_user":
```

Comment:
Nit: Can we add a case for "assume_role" so make it a bit more clear that this applies to both instead of re-using the same string value?
```
case "iam_user", "assume_role":
```

We'd update this bit to use "assume_role" instead.

https://github.com/hashicorp/vault/blob/86c1cdf9e61897dee14203c51d6ad3d971cfda1a/builtin/logical/aws/secret_access_keys.go#L208

Reply:
@calvn added!